stories/banner.stories.js

```diff
@@ -0,0 +1,267 @@
+import React from 'react';
+import { Story, Demo, Description, Code, comment } from './storybook/blocks';
+import './banner.scss';
+
+const title = 'Banner';
+
+export default { title };
+
+export const defaultBanner = () => (
+  <Story>
+    <Demo>
+      <div role="alert" className="Manifold__Banner">
+        <div className="Manifold__BannerIcon">
+          <svg
+            aria-label="info"
+            role="img"
+            viewBox="0 0 24 24"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <g fill-rule="evenodd" clip-rule="evenodd">
+              <path
+                d="M12 3a9 9 0 100 18 9 9 0 000-18zM1 12C1 5.925 5.925 1 12 1s11 4.925 11 11-4.925 11-11 11S1 18.075 1 12z"
+              />
+              <path
+                d="M12 11a1 1 0 011 1v4a1 1 0 11-2 0v-4a1 1 0 011-1zM11 8a1 1 0 011-1h.01a1 1 0 110 2H12a1 1 0 01-1-1z"
+              />
+            </g>
+          </svg>
+        </div>
+        Generic information uses a basic message bar.
```

Comment:
One thing I learned with `display: flex` container is that the text needs a wrapper (maybe a `<p>` tag. Because if you had an `<a>` tag mid-sentence, this would get weird (we noticed that on UI)